### PR TITLE
VG-31-Routing-till-egna-API-endpoints-som-utf-r-logik-g-llande-eran-data-i-JSON-fil-GET-POST-etc-

### PR DESCRIPTION
**Pull request by Rishad11**
Closes related issue.

Added server.js file
Added new dependencies: Axiom, cors and express
Update apiService for crud

### CODE REVIEW

File: src/router/server.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import express from 'express';
+import axios from 'axios';
+import cors from 'cors';
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+const EXTERNAL_API_BASE_URL = 'https://api.escuelajs.co/api/v1/products';
```
Comment (marcusju23) [inline src/router/server.js]: We are using different API: `https://fakestoreapi.com`

https://fakestoreapi.com/docs

Comment (Rishad11) [reply src/router/server.js]: Oh yeah, I forgot to change that.

File: src/router/server.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-const EXTERNAL_API_BASE_URL = 'https://api.escuelajs.co/api/v1/products';
+const EXTERNAL_API_BASE_URL = 'https://fakestoreapi.com/products';
```
Comment (marcusju23) [inline src/router/server.js]: Set base URL to `https://fakestoreapi.com` and update relevant requests to match endpoints.

**Review by marcusju23 — APPROVED** [review]
Well done! 🙌

**Review by Danback — APPROVED** [review]
No conflicts, tested and looks good. Well done!

**Pull request merged**